src/intl/ja/page-wallets-find-wallet.json

```
@@ -106,7 +107,7 @@
   "page-find-wallet-new-to-wallets": "ウォレットは初めてですか？始めるための概要はこちらになります。",
   "page-find-wallet-new-to-wallets-link": "イーサリアムウォレット",
   "page-find-wallet-not-all-features": "これらの機能をすべて備えたウォレットはありません",
-  "page-find-wallet-not-endorsements": "このページに記載されているウォレットは公式の推奨ではなく、情報提供のみを目的として提供されています。 これらの説明はウォレット会社自身によって提供されています。私たちは、私たちの基準に基づいて、このページにプロダクトを追加します。",
+  "page-find-wallet-not-endorsements": "Wallets listed on this page are not official endorsements, and are provided for informational purposes only. Their descriptions have been provided by the wallet projects themselves. We add products to this page based on criteria in our",
```

Why was this reverted...

```suggestion
  "page-find-wallet-not-endorsements": "このページに記載されているウォレットは公式の推奨ではなく、情報提供のみを目的として提供されています。 これらの説明はウォレット会社自身によって提供されています。私たちは、私たちの基準に基づいて、このページにプロダクトを追加します。",
```